magicctapipe/scripts/lst1_magic/lst1_magic_create_irf.py

```diff
@@ -592,7 +592,7 @@ def create_irf(
     # Save the data in an output file:
     Path(output_dir).mkdir(exist_ok=True, parents=True)
 
-    regex = r'dl2_gamma_(\S+)_run.*'
+    regex = r'dl2_(\S+)_run.*'
```

Comment:
this is also because in the "Test" production files there is no explicit "gamma"

Reply:
Here I changed the lines so that the script does not get the information from the input file name, but reads the pointing direction from the input data and use it for the output file name